internal/restapi/browse/frontend/src/data/useReleaseOrgsList.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+import dummyReleaseOrgs from '../assets/release-orgs-list.json'
+
+const dummyReleaseOrgResponseData = {
+    "releaseOrgs": dummyReleaseOrgs,
+}
+
+export interface ReleaseOrgResponse {
+    releaseOrgs: string[]
+}
+
+export const useReleaseOrgList = (): string[] => {
+    const releaseOrgListURI = "/api/release/organizations"
+    const [releaseOrgs, setReleaseOrgs] = useState<string[]>([])
+    useEffect(() => {
+        fetch(releaseOrgListURI)
+            .then((response) => {
+                return dummyReleaseOrgResponseData
```

Comment:
Why are we returning dummy responses for these?

Reply:
Have fixed it